llama-index-core/llama_index/core/schema.py

```diff
@@ -537,18 +539,25 @@ def hash(self) -> str:
 
 
 class Node(BaseNode):
-    text: MediaResource | None = Field(
+    text_resource: MediaResource | None = Field(
```

Comment:
Will this have an impact on the current openai multimodal stuff we had working?

Reply:
I think it won't, even if the logic is almost duplicated (and we might want to refactor it later), that depends on `ChatMessage` and not `Node`.